### ISSUE TITLE
4.8 release being used in the quay.io/openshift/okd-content image for 4.9

### ISSUE THREAD

**Issue by karthik101** [openshift/okd#1003]
**Describe the bug**

I run the install on centos 7 helper VM, and because of libvirt dependecy i used the openshift-install image, but for lastest 4.9 installer i see errors in the bootstrap VM, see below errors from the bootstrap vm

```image=registry.ci.openshift.org/origin/release@sha256:741c6e0ac31cd136f6b5cc739327d4ba958f5d175cce9a6fbf772fd97b5c5cfc, name=wonderful_curran, **io.openshift.release=4.8.0-0.okd-2021-12-01-110653**, io.openshift.release.base-image-digest=sha256:9cfeba63b06c2ebf4037d6e0362ca4e58259e7120d950904253c401d9004948a) Dec 01 17:18:43 dhcp-10-131-238-60 bootkube.sh[27138]: Rendering Kubernetes API server core manifests... Dec 01 17:18:43 dhcp-10-131-238-60 bootkube.sh[29155]: Error: error creating container storage: the container name "kube-apiserver-render" is already in use by "2fca7bc2287060741ebe638c8d917390b0831b50cf05be68fc87e68ac2d736a8". You have to remove that container to be able to reuse that name.: that name is already in use Dec 01 17:18:43 dhcp-10-131-238-60 systemd[1]: bootkube.service: Main process exited, code=exited, status=125/n/a Dec 01 17:18:43 dhcp-10-131-238-60 systemd[1]: bootkube.service: Failed with result 'exit-code'.```

The openshift-install image is picking the 4.8 release instead of 4.9. I checked the release version being used by openshift-install command inside the image, see below

For comparision i included the outputs from 4.8 installer image as well
```
[root@ocp4-vsphere]# /bin/oc version
Client Version: 4.9.0-0.okd-2021-11-28-035710

[root@ocp4-vsphere]# /bin/oc adm release info quay.io/openshift/okd:4.8.0-0.okd-2021-10-24-061736 --image-for=installer
quay.io/openshift/okd-content@sha256:e683c36b9b97f31136fbc4341912aabaa61001679978345be1e73e366fdf142e

[root@ocp4-vsphere]# docker run -v /root/karthik/ocp4-vsphere/install-dir/:/output:Z -t --user 0 quay.io/openshift/okd-content@sha256:e683c36b9b97f31136fbc4341912aabaa61001679978345be1e73e366fdf142e version
/bin/openshift-install unreleased-master-4706-g7b10e34a03fcd5df135ebeec314ea0a57e34c689
built from commit 7b10e34a03fcd5df135ebeec314ea0a57e34c689
release image registry.ci.openshift.org/origin/release:4.8
release architecture amd64
```

And then see release version from 4.9 installer image
```
[root@ocp4-vsphere]# /bin/oc adm release info quay.io/openshift/okd:4.9.0-0.okd-2021-11-28-035710 --image-for=installer
quay.io/openshift/okd-content@sha256:e76dbb3fced2060cbcf9acf52edf11f63f42a4ef2add8f4e1d0ee6ee41e23bf5

[root@ocp4-vsphere]# docker run -v /root/karthik/ocp4-vsphere/install-dir/:/output:Z -t --user 0 quay.io/openshift/okd-content@sha256:e76dbb3fced2060cbcf9acf52edf11f63f42a4ef2add8f4e1d0ee6ee41e23bf5 version
/bin/openshift-install unreleased-master-4997-g1c538b8949f3a0e5b993e1ae33b9cd799806fa93-dirty
built from commit 1c538b8949f3a0e5b993e1ae33b9cd799806fa93
release image registry.ci.openshift.org/origin/release:4.8
release architecture amd64
```

**Version**
4.9.0-0.okd-2021-11-28-035710

UPI install, with some updated ansible scripts based on https://github.com/RedHatOfficial/ocp4-vsphere-upi-automation

**How reproducible**
100%, since we only need oc command to check

**Comment by LorbusChris** [comment]
@karthik101 to work around this you can override the release image that the installer uses manually with an env var:
```
export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=quay.io/openshift/okd:4.8.0-0.okd-2021-10-24-061736
```
That should work, please let me know if it doesn't.

**Comment by karthik101** [comment]
So i needed the 4.9 installer , I used below command to fetch the openshift-install binary. The binary works without any libvirt requirement errors. Thanks for the work around.

```
oc adm release extract quay.io/openshift/okd:4.9.0-0.okd-2021-11-28-035710 --tools
```

**Comment by karthik101** [comment]
This is fixed in the 4.9.0-0.okd-2021-12-12-025847 release, closing this.
